interfaces/webidl.idl

```diff
@@ -0,0 +1,43 @@
+typedef (Int8Array or Int16Array or Int32Array or
+         Uint8Array or Uint16Array or Uint32Array or Uint8ClampedArray or
+         Float32Array or Float64Array or DataView) ArrayBufferView;
+
+typedef (ArrayBufferView or ArrayBuffer) BufferSource;
+[
```

Comment:
Okay, but then you should fix the IDL index at the end of the spec, because that's where I copied it from.  :)